website/docs/r/nsxt_security_group.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_security_group"
+sidebar_current: "docs-vcd-resource-nsxt-security-group"
+description: |-
+Provides a resource to manage NSX-T Security Group. Security groups are groups of data center
+group networks to which distributed firewall rules apply. Grouping networks helps you to reduce
+the total number of distributed firewall rules to be created.
+---
+
+# vcd\_nsxt\_security\_group
+
+Provides a resource to manage NSX-T Security Group. Security groups are groups of data center group
+networks to which distributed firewall rules apply. Grouping networks helps you to reduce the total
+number of distributed firewall rules to be created.
+
+Supported in provider *v3.3+* and VCD 10.1+ with NSX-T backed VDCs.
+
+## Specific usage notes
```

Comment:
This section is empty.

Reply:
Indeed. Left it here as I moved the text to Member VMs section:
```
~> There may be cases where Org Networks and Security Groups are already created, but
not all VMs are already created and not shown in this structure. Additional `depends_on` can ensure
that Security Group is created only after all networks and VMs are there.

```

Fixed.